fm2prof/MaskOutputFile.py

```diff
@@ -20,68 +20,72 @@
 Stichting Deltares and remain full property of Stichting Deltares at all times.
 All rights reserved.
 """
-
-import os
-
+from pathlib import Path
+from typing import Union, Optional
 import geojson
 
 
+
 class MaskOutputFile:
     @staticmethod
-    def create_mask_point(coords: geojson.coords, properties: dict):
+    def create_mask_point(coords: geojson.coords, properties: Optional[dict] = None) -> geojson.Feature:
         """Creates a Point based on the properties and coordinates given.
 
         Arguments:
             coords {geojson.coords} --
                 Coordinates tuple (x,y) for the mask point.
             properties {dict} -- Dictionary of properties

```

Comment:
```suggestion
            properties {dict} -- Dictionary of properties
            properties {dict, optional} -- Dictionary of properties. Defaults to None.
```

Reply:
The contributing page of the documentation suggets to use google style docstrings when writing new code. The docstrings throughout the code are not consistent at the moment. I'm working on a PR that will address this issue.